lldb/test/API/lang/swift/typealias_expression_parser/TestTypealiasExpressionParser.py

```diff
@@ -2,4 +2,4 @@
 from lldbsuite.test.decorators import *
 
 # FIXME! The Swift driver insists on passing -experimental-skip-non-inlinable-function-bodies-without-types to -emit-module.
-lldbinline.MakeInlineTest(__file__, globals(), decorators=[swiftTest, expectedFailureDarwin(bugnumber="rdar://120928396")])
+lldbinline.MakeInlineTest(__file__, globals(), decorators=[swiftTest, skipIf(bugnumber="rdar://120928396")])
```

Comment:
```suggestion
lldbinline.MakeInlineTest(__file__, globals(), decorators=[swiftTest, expectedFailureAll(bugnumber="rdar://120928396")])
```

Reply:
That was how I originally had it, but then it UPASSed on Ubuntu 20.04 https://ci.swift.org/job/pr-apple-llvm-project-linux/4333/

Reply:
Right, and that is probably how this got in in the first place.